pkg/report/pb/data-feeds/on-chain/registry/decode.go

```diff
@@ -1,74 +1,17 @@
 package registry
```

Comment:
This file was specifically put on a path `pkg/monitoring/pb/data-feeds/on-chain/registry` as this is the package to specifically deal with "monitoring/pb" (monitoring protobufs decoding), specifically the "beholder_domain" attr as "data-feeds/on-chain/registry" - does this makes sense? Please move it back and structure all Beholder monitoring packages for EVM, Aptos and every other chain like that, until a better proposal is documented and aligned.

Reply:
Moved this file back, will mimic the structure in the aptos PR as well

Reply:
Thank you!